app/src/main/java/lt/vilnius/tvarkau/NewProblemActivity.java

```diff
@@ -243,6 +246,7 @@ public void sendProblem() {
             Action1<Throwable> onError = throwable -> {
                 throwable.printStackTrace();
                 FirebaseCrash.report(throwable);
+                Crashlytics.logException(throwable);
```

Comment:
Consider abstracting logging to separate class `AppLog`. In this way we won't need to do call `FirebaseCrash::report` and `Crashlytics::logException` separately.